src/utilities/strategy-helpers/smi-canary-deployment-helper.ts

```diff
@@ -10,11 +10,13 @@ import * as TaskInputParameters from '../../input-parameters';
 import * as fileHelper from '../files-helper';
 import * as helper from '../resource-object-utility';
 import * as utils from '../manifest-utilities';
+import * as kubectlUtils from '../../kubectl-util';

```

Comment:
Why's the file two directories up? 
Add it to utilities folder

Reply:
It was in the same folder as run.ts and manifest-utilities.ts, because these were the only two files using kubectl-utils previously. Since smi-canary-deployment-helper.ts is also using it, I'll move it to utilities now.